modules/sdpolicy-security.adoc

```diff
@@ -58,6 +57,4 @@ See link:https://www.openshift.com/products/dedicated/process-and-security#compl
 
```

Comment:
Stupid github....

The line _above_ this comment should be updated with an `xref` that points to https://deploy-preview-34811--osdocs.netlify.app/openshift-dedicated/latest/osd_policy/policy-process-security.html#compliance_policy-process-security. This is because the docs.openshift.com version is going to replace the www.openshift.com version

Reply:
Stupid OpenShift docs...

We can't xref currently from within a module so the workaround is to name the module title here in plain text and then add a trailing include to the xref that will appear at the end of the module file by way of the parent assembly.  😞